### DISCUSSION TITLE
Message on admin login - This password appears in 7 data breaches! It is not unique; please change it.

### DISCUSSION THREAD

**Discussion by myownprovider** [Mailu/Mailu#3633] [User support] [answered]
Hi,
As the title mentions, when I login to the admin page I receive this message everytime I login to the admin portal with any of the admin id's.  I have checked with hibp site and it is not pwned.  I still changed all the passwords, but I think there is an issue with this.  There are no errors in the logs, so I don't know what to look at to troubleshoot.  There are no discussions about this, other than https://github.com/Mailu/Mailu/issues/2770.  
Thanks.

**Comment by zwa-vault** [comment]
Same here. I generated a fresh 32 char random password, but the message "This password appears in 7 data breaches" still appears.

**Comment by nextgens** [comment]
That's a bug. It will be fixed by https://github.com/Mailu/Mailu/pull/3650

**Reply by nextgens** [reply, marked as answer]
release 2024.06.26 has the fix
